uw-frame-components/static.html

```diff
@@ -29,13 +29,6 @@
         margin: -8px;
         font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;"
       id="loading-splash">
-        <!--Bad browser error message-->
-        <!--[if lt IE 10]>
-        <div class="bad-browser">
-          <span class="fa fa-frown-o"></span>
-          <p>Sorry, MyUW beta does not support your browser.<br/><a href="https://kb.wisc.edu/myuw/page.php?id=51345">Learn how to upgrade your browser.</a><br/>Can't upgrade? <a href="http://my.wisc.edu/portal/Login?profile=default">Switch back to MyUW classic.</a></p>
-        </div>
-        <![endif]-->
```

Comment:
That is the way these work. Knowledge drop: https://msdn.microsoft.com/en-us/library/ms537512(v=vs.85).aspx and http://www.quirksmode.org/css/condcom.html